### ISSUE TITLE
Downgrade CUDA SDK to 9.0

### ISSUE THREAD

**Issue by chfast** [ethereum-mining/ethminer#1390]
Currently we build binaries with CUDA 9.2. This is problematic because the ethminer does not work with a bit older Nvidia drivers. There are also no performance gains of using 9.2 over 9.0.

Downgrade the CUDA SDK used in CI builds to 9.0.

**Comment by smurfy** [comment]
Not sure but is it possible to support multiple cuda versions at once and then choose the kernel based on which version is available.

**Comment by chfast** [comment]
It's not about the kernel but about the cuda runtime library that is linked with the ethminer.

**Comment by smurfy** [comment]
Ah, ok. I'm all for compatibility, if performance is not affected. So by all means, change #1374 to be an downgrade instead.

**Comment by Zenitur** [comment]
I trying to compile ethminer 0.15 branch with CUDA 8.0.
```
[ 33%] Building NVCC (Device) object libethash-cuda/CMakeFiles/ethash-cuda.dir/ethash-cuda_generated_ethash_cuda_miner_kernel.cu.o
nvcc fatal   : Unsupported gpu architecture 'compute_70'
```
If remove "compute_70" from CMakeLists.txt: https://pastebin.com/xRDzPnRj

**Comment by AndreaLanfranchi** [comment]
@Zenitur CUDA 8.0 is not supported anymore.
You need CUDA 9 or better.

**Comment by jean-m-cyr** [comment]
No remaining issue here... closing

**Comment by XhmikosR** [comment]
@jean-m-cyr: I don't think this is done. https://github.com/ethereum-mining/ethminer/blob/master/scripts/install-cuda-ubuntu1604.sh#L13

**Comment by jean-m-cyr** [comment]
Ah sorry. Also CUDA 9.1 is fine. 9.2 introduced new driver dependencies.
